data/json/items/comestibles/med.json

```diff
@@ -1402,6 +1402,24 @@
     "addiction_type": "sleeping pill",
     "use_action": [ "FLUSLEEP" ]
   },
+  {
+    "id": "prophylactic_antivenom",
+    "type": "COMESTIBLE",
+    "comestible_type": "MED",
+    "name": { "str_sp": "prophylactic antivenom pill" },
+    "description": "A potent, cutting-edge antivenom in the form of some waxy white pills, designed to be taken in anticipation of venom exposure.  It was developed in top secret by the military in response to the advent of biological warfare.",
+    "weight": "1 g",
+    "volume": "250 ml",
+    "price": 10000,
+    "price_postapoc": 50000,
+    "charges": 20,
+    "stack_size": 200,
+    "symbol": "!",
+    "color": "white",
+    "healthy": -1,
+    "use_action": [ "PROPHYLACTIC_ANTIVENOM" ],
```

Comment:
You can use the consume_drug use action, scroll up to poppy painkiller to see an example. Duration can be specified as a string, as well.

Reply:
In that case, should I avoid fiddling with the effects in item_factory.cpp and item_actions.json?

Reply:
Yep, no C++ changes required then.